components/layouts/PopupLayout.tsx

```diff
@@ -14,6 +14,15 @@ export default function PopupLayout() {
   const location = useLocation();
   useResetPreline([location.pathname]);
 
+  // Resize the window to the target width and height for the popup
+  useEffect(() => {
+    const targetWidth = 375;
+    const targetHeight = 600;
+    const widthGap = targetWidth - window.innerWidth;
+    const heightGap = targetHeight - window.innerHeight;
+
+    window.resizeBy(widthGap, heightGap);
+  }, []);
```

Comment:
For those window created by browser-api, we need to fix window size since it is different between OS and browsers.

Reply:
Using `resizeTo` is not precise because it includes the window frame, not just the HTML content. We need to ensure that `innerWidth` and `innerHeight` match the target dimensions.

It is the result of using `resizeTo` in windows chrome.
![image](https://github.com/user-attachments/assets/14ff213e-152a-43de-b178-65494203f599)